backfill.sql

```diff
@@ -97,14 +101,16 @@ $$ LANGUAGE PLPGSQL VOLATILE;
 -- The main event
 -- staging_table is the (possibly temporary) table from which rows will be moved 
 -- destination_hypertable is the table where rows will be moved
--- on_conflict_do_nothing controls whether duplicate rows are ignored
+-- on_conflict_action controls whether duplicate rows are being ignored (DoNothing, default), an update clause is generated based on on_conflict_columns (Update), or if the duplicate row raises an error (Error) 
```

Comment:
This probably needs to be reworded, it's gotten a little long and hard to follow:
on_conflict action controls how duplicate rows are handled on insert, it has 3 allowed values that correspond to the ON CONFLICT actions in Postgres, 'NOTHING' (default), 'UPDATE', 'RESTRICT'. Their actions:
- NOTHING: ignore conflicting rows, the first inserted takes precedence
- UPDATE: replace values in the conflicting row according to the `on_conflict_update_columns` parameter, if this is set, the `on_conflict_update_columns` parameter must be set
- RESTRICT: error if there is a conflicting insert.